stac_mjx/io.py

```
@@ -12,11 +12,12 @@
 import h5py
 from pathlib import Path
 from omegaconf import DictConfig
-import os
+from jax.lib import xla_bridge
```

Ruff

`jax.lib.xla_bridge` imported but unused

Remove unused import: `jax.lib.xla_bridge`

(F401)